R/shared_constants.R

```diff
@@ -168,17 +172,14 @@ infix_metadata$unary <- infix_metadata$xml_tag %in% c("OP-PLUS", "OP-MINUS", "OP
 # high-precedence operators are ignored by this linter; see
 #   https://style.tidyverse.org/syntax.html#infix-operators
 infix_metadata$low_precedence <- infix_metadata$string_value %in% c(
```

Comment:
WDYT about moving infix_metadata to a csv file using sysdata.rda?

Reply:
my sense is it would bring more trouble than it's worth